.travis.yml

```diff
@@ -20,8 +20,12 @@ language: java
 jdk:
 - oraclejdk8
 
-install: cd tooltester && mvn -B clean install -DskipITs=false cobertura:cobertura coveralls:report
-
+install:
+- docker pull quay.io/garyluu/jenkins
+- docker network create --subnet=172.18.0.0/16 travisnet
```

Comment:
Sure, I meant comment in the source file so that future developers also know that